lib/githubModel.js

```
@@ -0,0 +1,53 @@
+      }
+    });
+
+    // Get the raw content from the model
+    const rawContent = response.data.choices[0].message.content.trim();
+    
+    // Clean the response if it's wrapped in markdown code blocks
+    let cleanedContent = rawContent;
+    if (rawContent.startsWith('```') && rawContent.endsWith('```')) {
+      // Remove markdown code block formatting
+      const match = rawContent.match(MARKDOWN_CODE_BLOCK_REGEX);
+      if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)